website/src/components/Devices.tsx

```diff
@@ -27,7 +28,7 @@ export const Devices = observer(
 
     render() {
       if (!this.devices.current) {
-        return <p>loading...</p>;
+        return <Loading />;
```

Comment:
There is still such a `<p>loading...</p>` string in the `AllDevices.tsx` file. Do we need to replace it as well?